src/controllers/notes.controller.js

```diff
@@ -0,0 +1,88 @@
+const { ne } = require('sequelize/dist/lib/operators')
+const Notes = require('../models/Notes')
+const Joi = require('joi')
+
+const getNotesSchema = Joi.object({
+    notes_id: Joi.string().guid().required(),
+    value: Joi.string().allow(''),
+})
+
+const createNotesSchema = Joi.object({
+    notes_id: Joi.string().guid().required(),
+    value: Joi.string().allow(''),
+})
+
+const updateNotesSchema = Joi.object({
+    id: Joi.string().guid().required(),
+    value: Joi.string().allow(''),
+})
+
+
+const get_notes = async(req, res, next)=>{
+    try{
+        const { value } = req.body
+       
+        const { error } = getNotesSchema.validate(notes_id, value)
```

Comment:
Te faltaron las llaves
```suggestion
        const { error } = getNotesSchema.validate({notes_id, value})
```